library/manageiq_tag_assignment.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/python
+
+import os
+from ansible.module_utils.basic import *
+from manageiq_client.api import ManageIQClient as MiqApi
+
+
+DOCUMENTATION = '''
+---
+module: manageiq_tag_assignment
+short_description: assign and unassign tags on resources in ManageIQ
+requirements: [ ManageIQ/manageiq-api-client-python ]
+author: Daniel Korn (@dkorn)
+options:
+  miq_url:
+    description:
+      - the manageiq environment url
+    default: MIQ_URL env var if set. otherwise, it is required to pass it
+  miq_username:
+    description:
+      - manageiq username
+    default: MIQ_USERNAME env var if set. otherwise, it is required to pass it
+  miq_password:
+    description:
+      - manageiq password
+    default: MIQ_PASSWORD env var if set. otherwise, it is required to pass it
+  tags:
+    description:
+      - list of dictionaries representing tags, each includes the tag name and category name
+      - assigning tags with the same category is not allowed in ManageIQ
```

Comment:
not quite true.  categories have a single_value: true/false.  assigning one tag in a single_value: true category clears other tag(s) from that category.